packages/flutter_tools/lib/src/web/chrome.dart

```diff
@@ -221,7 +221,7 @@ class ChromeLauncher {
       final HttpClientResponse response = await request.close();
       final List<dynamic> jsonObject = await json.fuse(utf8).decoder.bind(response).single as List<dynamic>;
       return base.resolve(jsonObject.first['devtoolsFrontendUrl'] as String);
-    } catch (_) {
+    } on Exception catch (_) {
```

Comment:
nit don't need to catch unused Exception

Reply:
Done